client/src/components/StudentCard.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import StudentCardCSS from "../css/StudentCard.module.css";
```

Comment:
Let's use object destructuring for cleaner JSX.

e.g. import { cardContainer, studentInfoCard, studentInfo, etc... } from <CSS FILE>